packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -7,6 +7,7 @@ const delays = {
   postResponse: 90 * 24 * 60 * 60 * 1000, // 90 days in ms
   postFirstDismissal: 7 * 24 * 60 * 60 * 1000, // 7 days in ms
   postSubsequentDismissal: 90 * 24 * 60 * 60 * 1000, // 90 days in ms
+  cooldown: 5 * 60 * 1000, // 5 minutes in ms
```

Comment:
I'm not too convinced by the name. What do you think about `preResponse` or `preUserInteraction` to match the formatting convention Rémi started? Or at least something that is a bit more explicit :)

Reply:
works for me